src/WebJobs.Extensions.DurableTask/EntityScheduler/RequestMessage.cs

```diff
@@ -43,6 +43,12 @@ internal class RequestMessage
         [JsonProperty(PropertyName = "parent", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string ParentInstanceId { get; set; }
 
+        /// <summary>
+        /// The parent instance that called this operation.
+        /// </summary>
+        [JsonProperty(PropertyName = "parentEx", DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
"Ex" as a suffix isn't super clear. I would suggest at least "parentExecution". I'm not too worried about the extra 7 characters.

Reply:
Sure.